draft-kinnear-webtransport-http2.md

```diff
@@ -168,6 +181,29 @@ stream to the destination endpoint. The only frames allowed on WebTransport
 Streams are WTHEADERS, CONTINUATION, DATA and any negotiated extension frames.
 
 
+# WebTransport Features
+
+WebTransport over HTTP/2 provides the following features described in
+[OVERVIEW]: unidirectional streams, bidirectional streams and datagrams,
+initiated by either endpoint.
+
+## Unidirectional streams
+
+Once a WebTransport session is established, either endpoint can initiate a
+unidirectional stream by sending a WTHEADERS frame with the Unidrectional flag
+set.
+
+## Bidirectional Streams
+
+Once a WebTransport session is established, either endpoint can initiate a
+bidirectional stream by sending a WTHEADERS frame.
+
+## Datagrams
+
+TBD
+
```

Comment:
This is a good start, I like the mini-section for each feature describing how to accomplish it with h2transport!